Smarty/templates/DocumentsListViewEntries.tpl

```diff
@@ -139,6 +139,9 @@
 															<input type="button" name="show" value="{$MOD.LBL_VIEW_EMPTY_FOLDERS}" class="crmbutton small cancel" onClick="fnvshobj(this,'emptyfolder');" title="{$MOD.LBL_VIEW_EMPTY_FOLDERS}">
 														</td>
 													{/if}
+													<td>
```

Comment:
Please convert this into a ListViewBasic link, somthing like this:
https://github.com/tsolucio/corebos/blob/master/build/changeSets/cbupdater_importxml.php#L26